GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -596,6 +606,7 @@ public Process GetGitProcess(string command, string workingDirectory, string dot
                         }
 
                         writeStdIn?.Invoke(this.executingProcess.StandardInput);
+                        this.executingProcess.StandardInput.Close();

```

Comment:
Why is this change necessary?

Reply:
`git pack-objects` listens to stdin until receiving an EOF. This is different from the other commands we've used in the past, which cared about an empty line.

Reply:
Got it. It's a harmless change for other commands but necessary for `git pack-objects`. Should we have a comment on why the line is needed?